translations/default/AdminShopparametersHelp.xlf

```diff
@@ -701,8 +701,8 @@
         <note>Line: 77</note>
       </trans-unit>
       <trans-unit id="3f141565d26297ff4449250743dba137">
-        <source>When active, new products will be activated by default during creation.</source>
-        <target>When active, new products will be activated by default during creation.</target>
+        <source>Enable this option if you want to activate by default all your manually created new products.</source>
+        <target>Enable this option if you want to activate by default all your manually created new products.</target>
```

Comment:
This is not needed, translation files are exported later during the release process ;)